test_conditionals.py

```diff
@@ -0,0 +1,24 @@
+import unittest2
+import main
+
+class TestConditionals(unittest2.TestCase):
+    def test_if(self):
+        js_code = 'if (true) {\n\tconsole.log("This is true")\n}'
+        py_code = 'if (True):\n\tprint("This is true")'
+        self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
+        self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+
+    def test_else(self):
+        js_code = 'if (1) {\n\tconsole.log("1 is true")\n} else {\n\tconsole.log("1 is NOT true")\n}' # TODO: consider adding ; after console.log()
+        py_code = 'if (1):\n\tprint("1 is true")\nelse:\n\tprint("1 is NOT true")\n'
+        self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
+        self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+   
+    def test_elif(self):
+        js_code = 'if (1) {\n\tconsole.log("1 is true")\n} else if (2) {\n\tconsole.log("2 is true")\n\tconsole.log("second line")\n}'
+        py_code = 'if (1):\n\tprint("1 is true")\nelif (2):\n\tprint("2 is true")\n\tprint("second line")\n'
+        self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
+        self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+
```

Comment:
Does a combination of `if/else if/else` work? If so mind adding a test case for that.

Reply:
Yes it does- just added in a test!